src/chainstate/stacks/boot/mod.rs

```diff
@@ -739,7 +907,12 @@ pub mod test {
         block_id: &StacksBlockId,
     ) -> Result<Vec<(StacksAddress, u128)>, Error> {
         let burn_block_height = get_par_burn_block_height(state, block_id);
-        state.get_reward_addresses(burnchain, sortdb, burn_block_height, block_id)
+        state
+            .get_reward_addresses(burnchain, sortdb, burn_block_height, block_id)
+            .and_then(|mut addrs| {
+                addrs.sort_by_key(|k| k.0.bytes.0);
```

Comment:
This means that addresses with more leading 0's are prioritized for PoX payouts.  Is that what we want?  Do we instead want to sort and then permute using the reward cycle block's VRF seed or something, to avoid creating an incentive for address grinding?

Reply:
This only sets the indexes used for the subsequent random choosing (implemented by the sortition db: each “next” address is chosen using the VRF seed). It _is_ important for consensus that this ordering is the same for all nodes, but it doesn’t impact the order of payouts in a predictable way.

Reply:
Ah, right, I'd forgotten that this was handled elsewhere.  Closing...